weather.go

```diff
@@ -76,7 +73,7 @@ func CallUrl(token, location string) http.Response {
 		}
 
 		if cu.Message == "city not found" {
-			log.Print("The city cannot be found")
+			log.Fatal("The city cannot be found")
```

Comment:
Libraries don't panic, and that includes calling `log.Fatal`—so we should perhaps return `error` from this function instead.

Reply:
fixed